common/src/Kokkos_ArithTraits.hpp

```diff
@@ -22,6 +22,10 @@
 
 #include <KokkosKernels_ArithTraits.hpp>
 
+#ifndef KOKKOS_ENABLE_DEPRECATED_CODE_5
+static_assert(false, "this header is deprecated, include <KokkosKernels_ArithTraits.hpp> instead");
```

Comment:
```suggestion
static_assert(false, "Header `Kokkos_ArithTraits.hpp` is deprecated, include `KokkosKernels_ArithTraits.hpp` instead");
```

I think it is better to explicitly print the name of this header